upstart/reddit-consumer-search_q.conf

```diff
@@ -10,5 +10,5 @@ respawn limit 10 5
 nice 10
 script
     . /etc/default/reddit
-    wrap-job paster run --proctitle cloudsearch_q$x $REDDIT_INI $REDDIT_ROOT/r2/lib/cloudsearch.py -c 'run_changed()'
+    wrap-job paster run --proctitle cloudsearch_q$x $REDDIT_INI -c 'from pylons import g; g.search.run_changed()'
```

Comment:
It should be safe to change `--proctitle cloudsearch_q$x` to `--proctitle search_q$x`.

@rram: The `proctitle` is just informational, right?